From 185bf38642ba3fd90c247cd9e37c45ff3d2be0d4 Mon Sep 17 00:00:00 2001
From: Mecid <mecid@mecomediagroup.de>
Date: Sun, 3 Aug 2025 20:43:14 +0200
Subject: [PATCH] Rockchip: Change boot order to nvme before emmc

Many boards have soldered emmc but optional m.2 slots. If NVMe has a boot target it's likely the user wants to boot of it instead of emmc.
---
 include/configs/rockchip-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/rockchip-common.h b/include/configs/rockchip-common.h
index d5550a46575..fb7f120eb99 100644
--- a/include/configs/rockchip-common.h
+++ b/include/configs/rockchip-common.h
@@ -14,7 +14,7 @@
 #ifndef CONFIG_XPL_BUILD
 
 #ifndef BOOT_TARGETS
-#define BOOT_TARGETS	"mmc1 mmc0 nvme scsi usb pxe dhcp spi"
+#define BOOT_TARGETS	"mmc1 nvme mmc0 scsi usb pxe dhcp spi"
 #endif
 
 #ifdef CONFIG_ARM64